rxtools/source/main.c

```diff
@@ -42,15 +42,15 @@ void Initialize(){
 		u32 pad = GetInput();
 		if(pad & BUTTON_R1 && i > 0x333333) goto rxTools_boot;
 	}
-    rxModeQuickBoot();
+	rxModeQuickBoot();
```

Comment:
I suggest to make some text before quickboot, also some after it.
So when it failed you can be able to know when.